src/docker-images/reaper/main.py

```diff
@@ -33,7 +33,11 @@ def kill():
             logger.info("processing alert of %s", alert)
             if not dry_run:
                 job_name = alert["labels"]["job_name"]
-                params = {"jobId": job_name, "userName": "Administrator"}
+                params = {
+                    "jobId": job_name,
+                    "userName": "Administrator",
+                    "desc": "killed because the job is being idle too long"
```

Comment:
Is it possible to expose the max GPU idle hours to users in the message?

Reply:
Yes, will need to add label to alert and receive label here.